DQM/BeamMonitor/plugins/AlcaBeamMonitorClient.cc

```diff
@@ -70,6 +70,8 @@ void AlcaBeamMonitorClient::beginRun(const edm::Run& r, const EventSetup& contex
 void AlcaBeamMonitorClient::analyze(const Event& iEvent, const EventSetup& iSetup) {}
 
 //----------------------------------------------------------------------------------------------------------------------
+void AlcaBeamMonitorClient::beginLuminosityBlock(const LuminosityBlock& iLumi, const EventSetup& iSetup) {}
```

Comment:
Not related to this line, but shouldn't the `~AlcaBeamMonitorClient() {}` be changed to  `~AlcaBeamMonitorClient() = default;` ?

Reply:
the same comment can be applied to 
- BeamConditionsMonitor.cc
- PixelVTXMonitor.cc
- Vx3DHLTAnalyzer.cc

Reply:
good point, I'll fix that too